src/jit/compile.c

```diff
@@ -123,7 +139,11 @@ MVMJitCode * MVM_jit_compile_graph(MVMThreadContext *tc, MVMJitGraph *jg) {
 }
 
 void MVM_jit_destroy_code(MVMThreadContext *tc, MVMJitCode *code) {
+#ifdef HAVE_LIBFFI
+    ffi_closure_free(code->writable_memory);
+#else
     MVM_platform_free_pages(code->func_ptr, code->size);
+#endif
```

Comment:
Does the exec memory not have to be freed also, or does `ffi_closure_free` take care of that?